common-util/Login/config.jsx

```diff
@@ -29,4 +30,8 @@ export const wagmiConfig = defaultWagmiConfig({
   metadata,
   ssr: true,
   storage: createStorage({ storage: cookieStorage }),
+  transports: SUPPORTED_CHAINS.reduce(
```

Comment:
this config should not be localized to a `Login` folder.. 
PascalCase is for components, pages, .etc

Reply:
I actually was really struggling holding a huge desire to refactor it all.... do you think we can postpone it until we migrate this to monorepo?